preferencesfx/src/main/java/com/dlsc/preferencesfx/model/PreferencesFxModel.java

```diff
@@ -53,7 +53,7 @@
   private boolean persistWindowState = false;
   private boolean saveSettings = true;
   private boolean historyDebugState = false;
-  private boolean oneCategoryLayout;
+  private boolean isOneCategoryLayout;
```

Comment:
Here, we chose the convention among all booleans to be named without `is` as the prefix and the corresponding method to be the prefix `is` with the name of the boolean appended